src/main/java/org/stellar/sdk/AccountConverter.java

```diff
@@ -0,0 +1,41 @@
+package org.stellar.sdk;
+
+import org.stellar.sdk.xdr.CryptoKeyType;
+import org.stellar.sdk.xdr.MuxedAccount;
+
+public class AccountConverter {
+  private final boolean enableMuxed;
+  private AccountConverter(boolean enabled) {
+    this.enableMuxed = enabled;
+  }
+
+  public static AccountConverter enableMuxed() {
+    return new AccountConverter(true);
+  }
+
+  public static AccountConverter disableMuxed() {
+    return new AccountConverter(false);
+  }
+
+
+  public MuxedAccount encode(String account) {
```

Comment:
This converts an input `account` string in both G and M-form into a `MuxedAccount` with only the G-account representation, right? (that is, forcing the Ed25519 discriminant). A docstring here would be :100: